CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # CHANGELOG
 
+
+## 3.2.0
```

Comment:
Do we follow [semver](https://semver.org) on BambooSMTP? If so, it should be the version `4.0.0`, no? 🤔

Reply:
Yep this seems fair enough. I will change the branch name and create new prs